README.md

```diff
@@ -1,6 +1,8 @@
 <p align="center"><img src="./readme_images/banner.png" width=500 /></p>
 
-**ArXiv Digest and Personalized Recommendations using Large Language Models.**
+**ArXiv Digest (extra version) and Personalized Recommendations using Large Language Models.**
+
+*(Note: This is an adjusted repo to match my needs. For original repo please refer to **AutoLLM** that I forked from)*
```

Comment:
This is a pull request to the original repo 😄

Reply:
Sorry Richard, pls ignore haha.